docs/transpile/representing_quantum_computers.mdx

```diff
@@ -113,14 +111,15 @@ Every quantum backend supports a limited instruction set, called its _basis gate
 The default single-qubit gates on _ibm_sherbrooke_ are `rz`, `x`, and `sx`, and the default two-qubit gate is `ecr` which standes for echoed cross resonsance. CX gates are constructed from `ecr` gates, so on some backends `ecr` is specified as the two-qubit basis gate while on others `cx` is default. The `ecr` gates is the _entangling_ part of the `cx` gate. If one desires to use a gate that is not in the basis gate set, instructions for custom gates can be provided using pulse gates (*link*). In addition to the control gates, there is also `delay` and `measurement` instructions.
 
 **question: the statement "the default single-qubit gates..." makes it sound like you can choose a different set of gates somehow.  That doesn't seem possible, so what does that statement mean?**
+Technically you can choose whatever gates you like as long as you provide the instruction or add pulse gates (see custom transpiler pass doc). The default basis gates are those that calibrations have been done for on the backend, so no further instrcution/pulse gate needs to be provided. For example, on some backends `cx` is the default two-qubit gates and `ecr` on others.
```

Comment:
thanks for that explanation.  Should we leave in any of that information or just remove it all?

Reply:
Yeah I think we can work that info into the above paragraph